src/splunk_otel/distro.py

```diff
@@ -32,3 +33,18 @@ def _configure(self, **kwargs):
     def set_env_defaults(self):
         for key, value in DEFAULTS.items():
             self.env.setdefault(key, value)
+
+    def load_instrumentor(self, entry_point, **kwargs):
```

Comment:
Perhaps a `FIXME` or similar comment here indicating that `OTEL_METRICS_ENABLED` should, by its very name, exist and be controlled by logic in the upstream sdk rather than here?

Reply:
Nevermind, I see the purpose of this bit now.